src/common/thread.cpp

```diff
@@ -55,16 +57,19 @@ bool Thread::createThread(const UString &name) {
 	_name = name;
 
 	// Try to create the thread
-	if (!(_thread = SDL_CreateThread(threadHelper, _name.empty() ? 0 : _name.c_str(), static_cast<void *>(this))))
+	try {
+		_thread = std::thread(threadHelper, static_cast<void *>(this));
```

Comment:
Aw, we're using the capability of naming our thread here. With `SDL_CreateThread`, the name would appear in the debugger, process monitors, etc.

But it seems like `std::thread` just doesn't have that functionality. Guess we're going to have to live with that